examples/Feeds/RemoveFlightsFeedItemAttributeValue.cs

```diff
@@ -0,0 +1,307 @@
+        {
+            // Get the GoogleAdsService.
+            GoogleAdsServiceClient googleAdsService = client.GetService(
+                Services.V4.GoogleAdsService);
+
+            // Constructs the query.
+            string query = $"SELECT feed.attributes FROM feed WHERE feed.resource_name = " +
+                           $"'{feedResourceName}'";
+
+            // Constructs the request.
+            SearchGoogleAdsRequest request = new SearchGoogleAdsRequest()
+            {
+                CustomerId = customerId.ToString(),
+                Query = query
+            };
+
+            // Issues the search request and get the first result, since we only need the
+            // single feed item we created previously.
+            GoogleAdsRow googleAdsRow = googleAdsService.Search(request).First();
+
+            // Gets the attributes list from the feed and creates a map with keys of each attribute
+            // and values of each corresponding ID.
+            Dictionary<FlightPlaceholderField, FeedAttribute> feedAttributes =
+                new Dictionary<FlightPlaceholderField, FeedAttribute>();
+
+            // Loops through the feed attributes to populate the map.
+            foreach (FeedAttribute feedAttribute in googleAdsRow.Feed.Attributes)
+            {
+                switch (feedAttribute.Name)
+                {
+                    case "Flight Description":
+                        feedAttributes[FlightPlaceholderField.FlightDescription] = feedAttribute;
+                        break;
+
+                    case "Destination ID":
+                        feedAttributes[FlightPlaceholderField.DestinationId] = feedAttribute;
+                        break;
+
+                    case "Flight Price":
+                        feedAttributes[FlightPlaceholderField.FlightPrice] = feedAttribute;
+                        break;
+
+                    case "Flight Sale Price":
+                        feedAttributes[FlightPlaceholderField.FlightSalePrice] = feedAttribute;
+                        break;
+
+                    case "Final URLs":
+                        feedAttributes[FlightPlaceholderField.FinalUrls] = feedAttribute;
+                        break;
+                    // The full list of FlightPlaceholderFields can be found here
+                    // https://developers.google.com/google-ads/api/reference/rpc/latest/FlightPlaceholderFieldEnum.FlightPlaceholderField
+                    default:
+                        throw new Exception("Invalid attribute name.");
+                }
+            }
+
+            return feedAttributes;
+        }
+
+        private FeedItem RemoveAttributeValueFromFeedItem(GoogleAdsClient client, in long
+                customerId,
+            Dictionary<FlightPlaceholderField, FeedAttribute> placeholdersToFeedAttributesMap,
+            string feedItemResourceName, FlightPlaceholderField flightPlaceholderFieldName)
```

Comment:
Could we modify this function to accept feedItemId and build the resourcename in the funtion instead? Same thoughts for wherever we can afford to do that in this example.

Reply:
Fixed.